### ISSUE TITLE
Create a "Settings screen"

### ISSUE THREAD

**Issue by giolaq** [AmazonAppDev/react-native-multi-tv-app-sample#20]
**Is your feature request related to a problem? Please describe.**
It would be nice to have a "Settings screen" accessible from the Account or Profile Icon of the main screen. The "Settings" screen can contain all the configurations related to a specific account for the app. Example: Switch theme, Language configuration and so on.

**Additional context**
Happy to discuss a design, we can be inspired by common TV apps setting screens.

**Comment by giolaq** [comment]
Hey @madhushree007 can I assign this to you? Put the screenshots here and let's discuss the implementation

**Comment by madhushree007** [comment]
Yes, Sure. Please assign this to me. I will put screenshot here and we can take it from there.

**Comment by madhushree007** [comment]
@giolaq Here is the screenshot from tvOS simulator.

<img width="1374" alt="Screenshot 2024-10-18 at 18 56 39" src="https://github.com/user-attachments/assets/148976f3-d962-40f2-ae16-ccf1cec9a038">
<img width="1333" alt="Screenshot 2024-10-18 at 18 56 53" src="https://github.com/user-attachments/assets/47eb4b00-19dd-40de-99a2-12894f9c1c3a">

**Comment by madhushree007** [comment]
@giolaq I have separated the settings menu from the main drawer and have put it at the bottom of the left side drawer. I also created settings page in a way that more settings feature can be integrated in this page. At the moment we only have Laaguage submenu. Please have a look. Thanks

![Simulator Screenshot - Apple TV - 2024-10-24 at 10 05 26](https://github.com/user-attachments/assets/09895ab3-c949-49f9-a0b7-a3f8049b4e35)
![Simulator Screenshot - Apple TV - 2024-10-24 at 10 05 02](https://github.com/user-attachments/assets/522b33a9-6860-4cf8-a067-2ea119623091)
![Simulator Screenshot - Apple TV - 2024-10-24 at 10 05 50](https://github.com/user-attachments/assets/1ff2d45a-40f2-41c9-95e7-63d9dcfd8310)

**Comment by giolaq** [comment]
This looks good! Thank you, I left some comment on the PR

**Comment by madhushree007** [comment]
@giolaq I did not find your comments in my PR. Could you please tag me with your comment.
